_shows/15_16/steady_rain.md

```diff
@@ -8,6 +8,8 @@ date_start: 2016-05-30
 date_end: 2016-05-31
 venue: New Theatre Studio A
 
+crew_incomplete: false
```

Comment:
External shows have 'ignore_missing' set to true automagically, so don't need crew_incomplete setting. Thoughts?

Reply:
I was unaware of this, I feel like it's nice to have for consistency